src/Okta.Sdk/Generated/GroupClient.Generated.cs

```diff
@@ -101,7 +101,7 @@ public Task<GroupRule> UpdateRuleAsync(GroupRule groupRule, string ruleId, Cance
         {
             Uri = "/api/v1/groups/rules/{ruleId}",
             Payload = groupRule,
```

Comment:
Not that this really matters, but we do not have the same line breaks here as we do in the others